src/css/fruits.css

```diff
@@ -0,0 +1,157 @@
+body {
+  background-color: #ae1332;
+}
+
+ul {
+  list-style: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+.fruits__container {
+  max-width: 375px;
+  padding-left: 20px;
+  padding-right: 20px;
+  margin: auto;
+}
+
+.fruits__list {
+  margin-top: 44px;
+  display: flex;
+  flex-wrap: wrap;
+  row-gap: 14px;
+}
+
+.fruits__title {
+  font-family: 'DM Sans', sans-serif;
+  font-weight: 500;
+  font-size: 28px;
+  line-height: 100%;
+  letter-spacing: -0.04em;
+  text-shadow: 0px 4px 70px rgba(0, 0, 0, 0.1);
+  text-transform: uppercase;
+  color: #ffffff;
+  text-align: center;
+  margin: 0;
+  padding-bottom: 14px;
+}
+
+.fruits__text {
+  font-family: 'DM Sans', sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 129%;
+  letter-spacing: -0.04em;
+  text-shadow: 0 4px 70px rgba(0, 0, 0, 0.1);
+  text-align: center;
+  color: #ffffff;
+  margin: 0;
+}
+
+.fruits__item {
+  width: 335px;
+  height: 465px;
+  border-radius: 20px;
+  text-align: center;
+}
+
+.fruits__item_strawberry {
+  background-color: #f8766b;
+}
+
+.fruits__item_banana {
+  background-color: #edd251;
+}
+
+.fruits__item_watermelon {
+  background-color: #9dc262;
+}
+
+.fruits__item_lime {
+  background-color: #bfd663;
+}
+
+.fruits__item_apple {
+  background-color: #e93f29;
+}
+
+.fruits__item_peach {
+  background-color: #db4f4b;
+}
+
+.fruits__subtitle {
+  font-family: 'DM Sans', sans-serif;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 117%;
+  letter-spacing: -0.04em;
+  color: #ffffff;
+  text-align: left;
+  padding-left: 24px;
+  margin-top: 64px;
+  margin-bottom: 8px;
+}
+
+.fruits__description {
+  font-family: 'DM Sans', sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 129%;
+  letter-spacing: -0.02em;
+  color: #ffffff;
+  text-align: left;
```

Comment:
text-align,padding-left i margin niepotrzebny. Lepiej to wszystko będzie ustawic w div dla samego h3 i p.